src/test/Fake.Core.UnitTests/Fake.DotNet.FxCop.fs

```diff
@@ -0,0 +1,323 @@
+module Fake.DotNet.FxCop
+
+open System
+open Fake.Core
+open Fake.DotNet
+open Fake.IO
+open Fake.IO.FileSystemOperators
+open Expecto
+
+let serializingObject = Object()
+
+let testCases =
+    if Environment.isWindows then
```

Comment:
Thanks a lot for the tests! I guess this is because fxcop doesn't run on non-windows platforms at all?

Reply:
That was my intention, yes.